_posts/2020-08-07-g2p.md

```diff
@@ -23,13 +23,13 @@ This blog post describes the background context for a software tool called `g2p`
 
 # What are the motivations behind G2P?
 
-There are many reasons why you might want to systematically convert between different characters[^1]. Here are a few possible use cases:
+There are many reasons why you might want to systematically convert between different characters. Here are a few possible use cases:
```

Comment:
The "movtivation" part of this blog post series should really make the reader feel like, by learning g2p, they can be awesome! They can create a readalong! They can solve their community's font-hack problem! They can create orthography converters! etc.

Reply:
Totally! This is a bit dry, I'll adjust this in the new blog posts